docs/Developers/Development_Setup.rst

```diff
@@ -43,14 +43,22 @@ Using virtualenv
 - Install virtualenv using pip :
     ::
 
-        $ pip install virtualenv
+        $ pip3 install virtualenv
 
 - Create the virtualenv :
     ::
 
         $ cd working_dir # move into the dir where you want to create coala-venv
         $ virtualenv coala-venv
 
+NOTE:
+If you have both Python 3 and Python 2 installed try this command,
```

Comment:
period at the end of this line please. (modification are advised because this sentence has become too long)